ipasetup.py.in

```diff
@@ -90,13 +90,14 @@ PACKAGE_VERSION = {
 common_args = dict(
     version=VERSION,
     license="GPLv3",
-    author="FreeIPA Developers",
+    author="freeIPA Developers",
     author_email="freeipa-devel@redhat.com",
-    maintainer="FreeIPA Developers",
+    maintainer="freeIPA Developers",
```

Comment:
Please perform the same change as in the previous PR.

Reply:
Let's do this in a separate ticket / PR